tests/unit/completion/test_completer.py

```diff
@@ -143,21 +168,17 @@ def _validate_cmd_prompt(cmd, txt):
     (':set general ignore-case |', usertypes.Completion.value),
     (':set general huh |', None),
     (':help |', usertypes.Completion.helptopic),
-    (':quickmark-load |', usertypes.Completion.quickmark_by_name),
-    (':bookmark-load |', usertypes.Completion.bookmark_by_url),
+    (':help     |', usertypes.Completion.helptopic),
     (':open |', usertypes.Completion.url),
-    (':buffer |', usertypes.Completion.tab),
-    (':session-load |', usertypes.Completion.sessions),
-    (':bind |', usertypes.Completion.empty),
+    (':bind |', None),
     (':bind <c-x> |', usertypes.Completion.command),
     (':bind <c-x> foo|', usertypes.Completion.command),
-    (':bind <c-x>| foo', usertypes.Completion.empty),
+    (':bind <c-x>| foo', None),
     (':set| general ', usertypes.Completion.command),
     (':|set general ', usertypes.Completion.command),
     (':set gene|ral ignore-case', usertypes.Completion.section),
     (':|', usertypes.Completion.command),
     (':   |', usertypes.Completion.command),
-    (':bookmark-load      |', usertypes.Completion.bookmark_by_url),
```

Comment:
They were redundant with some of the other test cases e.g. the fake
`bookmark-load` was testing pretty much the same thing as the fake
`quickmark-load` -- a command with a single positional argument with
completions.